metricq/history_client.py

```diff
@@ -497,8 +509,7 @@ async def history_aggregate_timeline(
             An iterator over aggregates for this metric.
 
         Raises:
-            InvalidHistoryResponse:
-                if an invalid response was received
+            ~exceptions.InvalidHistoryResponse:
```

Comment:
It seems redundant, but every raises had a short description for when the exception is raised. I'd add it for consistency

Reply:
It doesn't only seem redundant. It is redundant. It adds no useful information. It's the equivalent of

```
# Reset count to zero
count = 0
```

Really not a big fan.